source/administration/changelog.md

```diff
@@ -31,7 +31,7 @@ Mattermost v5.31.0 contains a low level security fix. [Upgrading](https://docs.m
  - Added a new ``manage_remote_clusters`` permission.
 
 ### Bug Fixes
- - Fixed an issue where a column in the ``ThreadMemberships`` table was missing.
+ - Fixed an issue where the ``UnreadMentions`` column was missing in the ``ThreadMemberships`` table for servers upgrading from an earlier version. Admins planning to enable [Collapsed Reply Threads](https://mattermost.com/blog/dev-sneak-peek-collapsed-reply-threads/) (available in beta in Q1 2021) are recommended to upgrade to v5.31.0 or later.
```

Comment:
...was missing in the ``ThreadMemberships`` table for servers upgrading from Mattermost Server v5.29.0